### PR TITLE
Update automotive-telemetry-analytics-content.md

### PR DESCRIPTION
**Pull request by msmarioo**
Updated with additional experiences and samples
Added reference to open source project in "Deploy this scenario"
Added 2 additional cost optimization and performance considerations.

### CODE REVIEW

File: docs/industries/automotive/automotive-telemetry-analytics-content.md [hunk 1]
```diff
@@ -133,14 +133,20 @@ This solution uses the following practices to help optimize costs:
...
 
 Performance efficiency is your workload's ability to scale efficiently to meet user demands. For more information, see [Performance efficiency pillar overview](/azure/architecture/framework/scalability/overview).
 
-In this workload, if the number and size of recorded data files is greater than 1,000 files or 300 MB a day, consider using Azure Batch with Azure Functions for decoding.
+- If the number and size of recorded data files is greater than 1,000 files or 300 MB a day, consider using Azure Batch for decoding.
+- Consider performing common calculations and analysis after ingest and storing them in additional tables.
+
+## Deploy this scenario
+
+To deploy Azure Data Explorer and ingest MDF files, you can follow the [step-by-step](https://github.com/microsoft/adx-automotive-demos/tree/main/mdf42adx) tutorial demonstrating how to deploy a free instance, parse MDF files, ingest and perform some basic queries.
```
Comment (ckittel) [inline docs/industries/automotive/automotive-telemetry-analytics-content.md:149]: This repo doesn't give you any demonstration on how to perform some basic queries.  Can you please include that information in the repo?

"Run queries on the data." is all that is said but doesn't help the learner out with any samples or interesting ones to run.  We don't like to link to content that "leaves the learner hanging" after they have deployed or configured something.  We want to show them how to use the thing you just had them deploy.  Please add that to the repo before we can link to it.

**Review by ckittel — CHANGES_REQUESTED** [review]

**Comment by ckittel** [comment]
Also, as an FTE, I'd be glad to show you how you can access our internal PR experience, which will give you a bit more support in the process.  Feel free to reach out to me for guidance.

**Comment by msmarioo** [comment]
Hi Chad,

Thanks for the feedback, I do have some queries to add - I will do so tomorrow morning so we can add this link.

I would also like to have access to the internal PR experience - is there docvumentation for it?

Thanks,
________________________________

**Comment by ckittel** [comment]
I've reached out to you on Teams to help you get access.

**Pull request closed without merging**